src/components/annotations/index.js

```diff
@@ -343,8 +343,8 @@ annotations.draw = function(gd, index, opt, value) {
         .classed('annotation-text-g', true)
         .attr('data-index', String(index));
 
-    var ann = anng.append('svg')
-        .call(Plotly.Drawing.setPosition, 0, 0);
+    var ann = anng.append('g')
+        .attr('transform', 'translate(0,0)');
```

Comment:
Yep - the call to `setPosition` only sets `x` and `y` attributes - for `<g>`'s, we need to position using `transform`.